ironfish/src/rpc/routes/workers/getStatus.ts

```diff
@@ -88,16 +89,11 @@ router.register<typeof GetWorkersStatusRequestSchema, GetWorkersStatusResponse>(
 function getWorkersStatus(node: IronfishNode): GetWorkersStatusResponse {
   const result: GetWorkersStatusResponse['jobs'] = []
 
-  for (const name of node.workerPool.stats.keys()) {
-    // Move control messages to top level message and not request body type
-    if (name === 'jobAbort' || name === 'sleep') {
-      continue
-    }
```

Comment:
We should probably keep these hidden, since they're mostly internal/testing messages (I'm not sure what the comment means, though).